packages/design-system-docs/src/pages/components/Tooltip/Tooltip.example.jsx

```diff
@@ -5,90 +5,97 @@ import ReactDOM from 'react-dom';
-        triggerContent="placement"
-        triggerClassName="ds-c-tooltip__trigger-link"
+        offset={[0, 25]}
+        title="Tooltip positioned with custom offset"
       >
-        Tooltip positioned on the right
+        Tooltip with custom offset
       </Tooltip>
     </div>
     <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-y--2">
-      <p className="ds-u-margin--0 ds-u-margin-right--1">Tooltip with custom</p>
       <Tooltip
-        offset={[0, 20]}
-        triggerContent="offset"
-        triggerClassName="ds-c-tooltip__trigger-link"
+        dialog
+        className="ds-c-button"
+        title={
+          <>
+            <p className="ds-u-margin--0">
+              Tooltip dialogs only activate on click and include a focus trap. Intended for tooltips
+              with complex layout and
+              <a className="ds-c-link--inverse" href="/#">
+                {' '}
+                interactive elements
+              </a>
+            </p>
+            <Button
+              size="small"
+              className="ds-u-margin-top--2"
+              href="https://dequeuniversity.com/library/aria/tooltip-dialog"
+            >
+              More info
+            </Button>
+          </>
+        }
       >
-        Tooltip positioned with custom offset
+        Tooltip dialog for interactive content
       </Tooltip>
     </div>
-    <div className="ds-u-display--flex ds-u-align-items--center ds-u-margin-y--2">
-      <p className="ds-u-margin--0 ds-u-margin-right--1">Tooltip dialog activated </p>
-      <Tooltip dialog triggerContent="on click" triggerClassName="ds-c-tooltip__trigger-link">
-        <>
-          <p className="ds-u-margin--0">
-            {
-              'Tooltip dialogs only activate on click and include a focus trap. Intended for tooltips with complex layout and multiple interactive elements '
-            }
-          </p>
-          <Button
-            size="small"
-            className="ds-u-margin-top--2"
-            href="https://dequeuniversity.com/library/aria/tooltip-dialog"
+    <div className="example--inverse example--wrapper">
+      <div className="ds-u-display--flex ds-u-align-items--center">
+        <p className="ds-u-margin--0">Inverse tooltip with icon trigger</p>
+        <Tooltip
+          activeClassName="ds-c-tooltip-icon--active"
+          ariaLabel="Label describing the subject of the inverse tooltip"
+          className="ds-c-tooltip__trigger-icon"
+          inversed
+          placement="right"
+          title="Inverse tooltip styles applied"
+        >
+          <TooltipIcon inversed />
+        </Tooltip>
+      </div>
+      <div className="">
+        <p className="ds-u-margin--0">
+          {'Inverse tooltip with '}
+          <Tooltip
+            className="ds-c-tooltip__trigger-link"
+            component="a"
+            inversed
+            title="Tooltip trigger uses <a> for the trigger, styled with dashed underline"
```

Comment:
Not sure if I'm looking in the right place since I'm new to the DS PRs, but the inversed inline trigger doesn't show dashed underline for me, even though the tooltip content says it's supposed to be: 
![image](https://user-images.githubusercontent.com/21293083/111470228-0219ea00-86fe-11eb-9342-897c6af54562.png)

Reply:
@kimieann @LouisFettet I filed [this ticket](https://jira.cms.gov/browse/WNMGDS-850) to capture the two styling issues that have been brought up

Reply:
I think it would be worthwhile to try and gain alignment between Medicare and Healthcare on tooltip link styling so we can implement consistently across CMS